Frontend/Controllers/Projects/AcademyPerformanceController.cs

```diff
@@ -11,23 +13,47 @@ namespace Frontend.Controllers.Projects
     public class AcademyPerformanceController : Controller
     {
         private readonly IGetInformationForProject _getInformationForProject;
+        private readonly IProjects _projectsRepository;
 
-        public AcademyPerformanceController(IGetInformationForProject getInformationForProject)
+        public AcademyPerformanceController(IGetInformationForProject getInformationForProject, IProjects projectsRepository)
         {
             _getInformationForProject = getInformationForProject;
+            _projectsRepository = projectsRepository;
         }
 
-        public async Task<IActionResult> Index(string id)
+        [HttpGet]
+        public async Task<IActionResult> Index(string id, bool addOrEditAdditionalInformation = false)
         {
             var projectInformation = await _getInformationForProject.Execute(id);
 
             var model = new AcademyPerformanceViewModel
             {
                 Project = projectInformation.Project,
-                OutgoingAcademy = projectInformation.OutgoingAcademy
+                OutgoingAcademy = projectInformation.OutgoingAcademy,
+                AdditionalInformationModel = new AdditionalInformationViewModel
+                {
+                    AdditionalInformation = projectInformation.Project.AcademyPerformanceAdditionalInformation,
+                    HintText = "This information will populate into your HTB template under the school performance (Ofsted information) section.",
+                    Urn = projectInformation.Project.Urn,
+                    AddOrEditAdditionalInformation = addOrEditAdditionalInformation
```

Comment:
It'd be good to add tests that the three non-static bits of information get set on the model correctly (`AdditionalInformation`, `Urn`, and `AddOrEditAdditionalInformation`)

There's a helper for getting view models in `ControllerTestHelpers` and then you can assert against the values there